### PR TITLE
[release-4.3] Bug 1876486: static-pod-installer: recreate installers that disappeared

### PR DESCRIPTION
**Pull request by soltysh**
The line

```
ret.TargetRevision = currNodeState.CurrentRevision
```

led to a dead-lock because the installer controller only does anything if either

- all `TargetRevision`s are zero
- or the `TargetRevision` value is larger than `CurrentRevision`.

We went into the second case and the controller dead-locked.

/assign @sttts

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -273,7 +273,7 @@ imports:
   - libcontainer/system
   - libcontainer/user
 - name: github.com/openshift/api
-  version: 371e06d3a8a73b95b67bf6d0b7fb6259c67a5e62
+  version: a843dc3262a0cdc5fe6c20620f3e47f5f905efd1
```
Comment (sttts) [inline glide.lock:276]: why was this bumped? Is it the right branch?

Comment (soltysh) [reply glide.lock:276]: yup, it's 4.3 https://github.com/openshift/library-go/blob/release-4.3/glide.yaml#L16

**Comment by sttts** [comment]
Overriding the valid-bug, because these are just the library PRs. The bumps will go the normal way.

**Pull request merged**